packages/@aws-cdk/aws-msk/README.md

```diff
@@ -84,8 +84,6 @@ const cluster = msk.Cluster.fromClusterArn(this, 'Cluster',
 
 [MSK supports](https://docs.aws.amazon.com/msk/latest/developerguide/kafka_apis_iam.html) the following authentication mechanisms.
 
-> Only one authentication method can be enabled.
-
```

Comment:
In this case I'd think that a replacement of text instead of a removal would work better.